examples/react-native-bare-example/src/screens/Home.tsx

```diff
@@ -0,0 +1,147 @@
+import React, {useCallback} from 'react';
+
+import {RNAlchemySigner} from '@account-kit/react-native-signer';
+import type {User} from '@account-kit/signer';
+import {useEffect, useState} from 'react';
+import {
+  View,
+  Text,
+  TextInput,
+  StyleSheet,
+  Linking,
+  TouchableOpacity,
+} from 'react-native';
+import {API_KEY} from '@env';
+
+const signer = new RNAlchemySigner({
+  client: {connection: {apiKey: API_KEY!}},
+});
+
+export default function HomeScreen() {
+  const [email, setEmail] = useState<string>('');
+  const [user, setUser] = useState<User | null>(null);
+
+  const handleUserAuth = ({bundle}: {bundle: string}) => {
+    signer
+      .authenticate({
+        bundle,
+        type: 'email',
+      })
+      .then(setUser)
+      .catch(console.error);
+  };
+
+  const handleIncomingURL = useCallback((event: {url: string}) => {
+    const regex = /[?&]([^=#]+)=([^&#]*)/g;
+
+    let params: Record<string, string> = {};
+    let match: RegExpExecArray | null;
+
+    while ((match = regex.exec(event.url))) {
+      if (match[1] && match[2]) {
+        params[match[1]] = match[2];
+      }
+    }
+
+    if (!params.bundle || !params.orgId) {
+      return;
+    }
+
+    handleUserAuth({
+      bundle: params.bundle ?? '',
+    });
+  }, []);
+
+  useEffect(() => {
+    // get the user if already logged in
+    signer.getAuthDetails().then(setUser);
+  }, []);
+
+  // Add listener for incoming links
+  useEffect(() => {
+    const subscription = Linking.addEventListener('url', handleIncomingURL);
+
+    return () => subscription.remove();
+  }, [handleIncomingURL]);
+
+  return (
+    <View style={styles.container}>
+      {!user ? (
+        <>
+          <TextInput
+            style={styles.textInput}
+            placeholderTextColor="gray"
+            placeholder="enter your email"
+            onChangeText={setEmail}
+            value={email}
+          />
+          <TouchableOpacity
+            style={styles.button}
+            onPress={() => {
+              signer.authenticate({email, type: 'email'}).catch(console.error);
+            }}>
+            <Text style={styles.buttonText}>Sign in</Text>
+          </TouchableOpacity>
+        </>
+      ) : (
+        <>
+          <Text style={styles.userText}>
+            Currently logged in as: {user.email}
+          </Text>
+          <Text style={styles.userText}>OrgId: {user.orgId}</Text>
+          <Text style={styles.userText}>Address: {user.address}</Text>
```

Comment:
if we create a LA client, then we should have two fields here:
1. Signer Address
2. Account Address